cli/src/commands/config/unset.rs

```diff
@@ -0,0 +1,36 @@
+use tracing::instrument;
+
+use super::ConfigLevelArgs;
+use crate::cli_util::get_new_config_file_path;
+use crate::cli_util::CommandHelper;
+use crate::command_error::user_error;
+use crate::command_error::CommandError;
+use crate::config::remove_config_value_from_file;
+use crate::config::ConfigNamePathBuf;
+
+/// Update config file to unset the given option.
+#[derive(clap::Args, Clone, Debug)]
+pub struct ConfigUnsetArgs {
+    #[arg(required = true)]
+    name: ConfigNamePathBuf,
+    #[command(flatten)]
+    level: ConfigLevelArgs,
+}
+
+#[instrument(skip_all)]
+pub fn cmd_config_unset(
+    command: &CommandHelper,
+    args: &ConfigUnsetArgs,
+) -> Result<(), CommandError> {
+    let config_path = get_new_config_file_path(&args.level.expect_source_kind(), command)?;
+    if config_path.is_dir() {
+        return Err(user_error(format!(
+            "Can't set config in path {path} (dirs not supported)",
+            path = config_path.display()
+        )));
+    }
+
+    // TODO(pylbrecht): do we need to check_wc_author() here?
```

Comment:
I saw we have this check here:
https://github.com/martinvonz/jj/blob/f4ec653e12ddaeb39c98124dad7d2de5845321d2/cli/src/commands/config/set.rs#L66-L72

Do we need to inform the user in the same way for `jj config unset`?

Reply:
Maybe no. It's technically correct that unsetting repo config may change the user name/email, but I expect such situation is rare.